smart_fact_crawler/__init__.py

```diff
@@ -128,7 +128,7 @@ def sipm_currents(url=smartfacturl + 'current.data'):
     tc = TableCrawler(url)
     return to_namedtuple('CurrentPage', {
         'timestamp': sft2dt(tc[0, 0]),
-        'calibrated': tc[1, 1],
+        'calibrated': tc[1, 1] == 'yes',
```

Comment:
This one is clever. It fits our mentality to convert every entry into its appropriate representation.